invesalius/gui/dialogs.py

```diff
@@ -4430,13 +4431,16 @@ def GetValue(self):
         return self.com_ports.GetString(self.com_ports.GetSelection()).encode(const.FS_ENCODE), fn_probe, fn_ref, fn_obj
 
 
-class SetCOMport(wx.Dialog):
-    def __init__(self, title=_("Select COM port")):
-        wx.Dialog.__init__(self, wx.GetApp().GetTopWindow(), -1, title, style=wx.DEFAULT_DIALOG_STYLE|wx.FRAME_FLOAT_ON_PARENT|wx.STAY_ON_TOP)
+class SetCOMPort(wx.Dialog):
```

Comment:
This is not needed for this purpose, so I recommend removing the UI for selecing baudrate. However, this might be useful for other serial communications in the software. I would maybe leave the code but without any direct use.

Reply:
Thanks, I disabled the configurable baud rate from the UI - but left in the option of bringing it back with a simple parameter change to the constructor of SetCOMPort.